### PR TITLE
Updated documentation for delivery attempt

### PR DESCRIPTION
**Pull request by hmvien**
Make sure you've checked off all these things before submitting:

- [ ] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [ ] This pull request is submitted to the `dev` branch.
- [ ] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.

### CODE REVIEW

File: source/includes/getting-started.md [hunk 1]
```diff
@@ -421,25 +421,31 @@ Great, we’ve got a route! What’s next?
...
+In the circumstances when a driver does not know when he/she should visit the waypoint again, it is recommended that you should mark the delivery as `DeliveryStatus.FAILED`. Later during the day, if the driver would like to
+go back to the waypoint, you should submit a `DeliveryStatus.AUTHORIZE_REATTEMPT`. Foxtrot will re-schedule waypoint to be visited later.
 
-The `deliveryCode` is useful for gathering structured driver feedback. For example, you may wish to have the driver choose a `deliveryCode` from a list of failure reasons. Using consistent `deliveryCode`s will allow your users to identify common issues and improve their operation.
+<aside class="notice">
+In order to submit a DeliveryStatus.VISIT_LATER or DeliveryStatus.AUTHORIZE_REATTEMPT, a different api should be used, FoxtrotSDK.markDeliveryToVisitLater() and FoxtrotSDK.authorizeDeliveryReattempt(), respectively.
+These two api require a DeliveryVisitLater or DeliveryReattempt, respectively. An optional TimeWindow can be set when constructing these objects if the driver know the expected time to visit the waypoint again.
```
Comment (loganakamatsu) [inline source/includes/getting-started.md]: nitpick: "These two api" -> "These two apis" and "the driver know" -> "the driver knows"

File: source/includes/models/user-created.md [hunk 2]
```diff
@@ -151,11 +187,28 @@ DeliveryCode deliveryCode = DeliveryCode.builder()
...
+Both `AUTHORIZE_REATTEMPT` and `VISIT_LATER` emit the same behavior; however, it is recommended to use them based on the
+context of the situation.
+
+### `VISIT_LATER` sample use-cases:
+
+1. Driver failed to deliver the goods product, but knows he/she will come back later during the day.
+
+### `AUTHORIZE_REATTEMPT` sample use-cases:
+
+1. Driver failed to delivery the goods and marked it as `FAILED`. Later during the day, driver wants to come back.
```
Comment (loganakamatsu) [inline source/includes/models/user-created.md]: nitpick: "driver failed to delivery" -> "driver failed to deliver"
"driver wants to come back" -> "the driver wants to come back"

File: source/includes/models/user-created.md [hunk 3]
```diff
@@ -151,11 +187,28 @@ DeliveryCode deliveryCode = DeliveryCode.builder()
...
+The driver can now `AUTHORIZE_REATTEMPT` on the same delivery. Foxtrot will automatically re-optimize the route with the
+new information.
+2. Driver failed to delivery the goods and marked it as `FAILED`. Later during the day, a route manager wants the driver
```
Comment (loganakamatsu) [inline source/includes/models/user-created.md]: nitpick: "driver failed to delivery" -> "driver failed to deliver"

**Comment by loganakamatsu** [comment]
Sorry to be a grammar grandma... definitely like the changes + they add a lot of clarity.

File: source/includes/getting-started.md [hunk 4]
```diff
@@ -456,7 +462,8 @@ And similarly, a failed attempt with a DeliveryCode explaining why the attempt w
 ```java
 DeliveryCode deliveryCode = DeliveryCode.builder()
                                         .setCode("F-605")
-                                        .setMessage("The customer is not open on Thursdays");
+                                        .setMessage("The customer is not open on Thursdays")
+                                        .build();
```
Comment (isaaczafuta) [inline source/includes/getting-started.md]: :+1:

**Pull request merged**